app/Http/Controllers/ProfileController.php

```diff
@@ -15,16 +16,18 @@
 use Illuminate\View\View;
 use SonarSoftware\CustomerPortalFramework\Controllers\AccountAuthenticationController;
 use SonarSoftware\CustomerPortalFramework\Controllers\ContactController;
-use SonarSoftware\CustomerPortalFramework\Models\Contact;
 use SonarSoftware\CustomerPortalFramework\Models\PhoneNumber;
+use SonarSoftware\CustomerPortalFramework\Models\Contact;
 
 class ProfileController extends Controller
 {
     private $passwordPolicy;
+    protected $contactService;
 
-    public function __construct()
+    public function __construct(ContactService $contactService)
```

Comment:
Similar to the `BillingController`, there are a number of navigation references that may be impacted by this like `->action([ProfileController::class, 'show'])`.